Watchman.Discord/Areas/Statistics/Controllers/StatisticsController.cs

```diff
@@ -24,6 +25,7 @@ public class StatisticsController : IController
         private readonly IQueryBus _queryBus;
         private readonly ICommandBus _commandBus;
         private readonly MessagesServiceFactory _messagesServiceFactory;
+        private readonly Period[] implementedBySplitter = new Period[] { Period.Day };
```

Comment:
Przywróciłbym komentarz, że to będzie do wywalenia w przyszłości

Reply:
potwierdzam 
powinna być ta informacja